tests/xtimer_now_irq/tests/01-run.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2020 Freie Universität Berlin,
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# @author      Julian Holzwarth <julian.holzwarth@fu-berlin.de>
+
+import sys
+from testrunner import run
+
+TIMEOUT = 75 * 60
```

Comment:
On non-32bit timers (next known size ios 24bit), shit should fail after 2**24 -> ~16 seconds, right?

Reply:
yes